core/src/rpc.rs

```diff
@@ -3119,24 +3122,27 @@ pub mod tests {
         bank.store_account(&address, &account);
 
         let req = format!(
-            r#"{{"jsonrpc":"2.0","id":1,"method":"getAccountInfo","params":["{}", {{"encoding":"binary64"}}]}}"#,
+            r#"{{"jsonrpc":"2.0","id":1,"method":"getAccountInfo","params":["{}", {{"encoding":"base64"}}]}}"#,
```

Comment:
Ah, this feels so much nicer.  "binary64"?  What's that!  "base64"?  Oh, that's base64.

Reply:
Yes yes yes!! 🙌